lib/GADS/Column/Person.pm

```diff
@@ -195,5 +211,44 @@ sub import_value
     });
 }
 
-1;
+sub values_beginning_with {
+    my ( $self, $match_string, %options ) = @_;
+
+    my $resultset = $self->resultset_for_values;
+    my @value;
+    my $value_field = 'me.' . $self->value_field;
+    $match_string =~ s/([_%])/\\$1/g;
+        my $search =
+      $match_string
+      ? {
+        $value_field => {
+            -like => "${match_string}%",
+        },
+      }
+      : $options{noempty}
+      && !$match_string ? { \"0 = 1" }    # Nothing to match, return nothing
+      :                   {};
+    if ($resultset) {
+        my $filter       = $self->filter;
+        my $match_result = $resultset->search(
+            $search,
+            {
+                rows => 10,
+            },
+        );
+        @value = map {
```

Comment:
This is where this implementation differs (aside from the inclusion of the filter above) - as this doesn't use grouping in the same manner as the original it is felt that this should remain here, rather than bastardising the original implementation